src/corner-algorithms.jl

```diff
@@ -145,3 +145,33 @@ function gammacovs(cov_xx, cov_xy, cov_yy, gamma::Float64 = 1.4)
 
     filt_cov_xx, filt_cov_xy, filt_cov_yy
 end
+
+
+"""
+```
+moravec_response = moravec(img; [threshold], [window_size])
+```
+Performs Moravec corner detection. 
+The Moravec Corner Detector algorithm calculates corner responses based on intensity differences.
+"""
+function moravec(img::AbstractArray; threshold::Float64 = 10000.0, window_size::Int = 3, args...)
+    gradient_x, gradient_y = gradcovs(img, args...)
```

Comment:
```suggestion
function moravec(img::AbstractArray, args...; threshold::Float64 = 10000.0, window_size::Int = 3)
    gradient_x, gradient_y = gradcovs(img, args...)
```

I think you actually want to forward the positional arguments, not the keyword arguments.